cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -24,7 +28,7 @@ def map_tags(appliance, provider, request):
 
 
 @pytest.fixture(scope='function')
-def vm(appliance, provider):
+def tagged_vm(provider):
```

Comment:
@mshriver No not related to changes.
I think we need to move it on any available `VM`. No need specific vm for tagging.  
Yup It take time now as UI but I am going to move `VM` `all` method with rest. If you add TODO(ndhandre) it will be great. Just request if possible.

Reply:
as per offline conversation resolving.